lib/themes/src/color-themes/ivv-navy.ts

```diff
@@ -0,0 +1,50 @@
+import { definePreset } from '@primeuix/themes';
+import { SudososPreset } from '../sudosos';
+
+export const IvvNavy = definePreset(SudososPreset, {
```

Comment:
Look I'm not I.V.V, but isn't navy a blue color? and isn't this Bordeaux?

Reply:
Apparently Bordeaux is Dutch, but the translation is Wine Red, which I think is pretty funny
https://nl.wikipedia.org/wiki/Bordeauxrood <-> https://en.wikipedia.org/wiki/Wine_red
https://nl.wikipedia.org/wiki/Marineblauw <-> https://en.wikipedia.org/wiki/Navy_blue